.gitignore

```diff
@@ -64,3 +67,8 @@ waves.shm/
 
 # Interface libraries built on setup
 cocotb/libs
+pip-wheel-metadata/
+
+# Mentor Modelsim/Questa
```

Comment:
`modelsim.ini and transcript are also in examples maybe block globally

Reply:
also `*.wlf`

Reply:
also some `*.json` files

Reply:
> `modelsim.ini` and `transcript` are also in examples maybe block globally

I think a user can add local ``modelsim.ini`` files to define libraries etc., so ignoring them globally may be bad.

Reply:
> also some `*.json` files

Globally, this may be too much. Is this specific to wavedrom maybe?